### PR TITLE
Infinera Groove: Add cardAdminStatus states

### PR DESCRIPTION
**Pull request by Bierchermuesli**
Adding `cardAdminStatus` state to discovery so we can check the real operational state of a module. If the `cardMode` is notApplicable (not normal nor regen) it will be skipped as it does not make much sense on my release. I used a G30 with 4.7.0. @nickhilliard has tested with 3.0.0 anno 2019

CORIANT-GROOVE-MIB says:

> cardAdminStatus:
> 	The operational state specifies whether or not a resource is able to provide service. 
> 	The operational state shall be visible to the operator. The user cannot modify the operational state. It provides a simple mechanism for the operator to decide whether a resource is operational or not, therefore this parameter has a read-only nature. 
> 	The operational state is closely coupled with the alarm status of a resource, i.e. state transitions are triggered internally by the NE software. It can have one of the following values:
> 	Down: The resource is totally inoperable and unable to provide service to the user(s)
> 	Up: The resource is partially or fully operable and available for use. 

> cardMode:
> Specify` the card mode, e.g. normal or regen.

Actually I don't get it what  `cardMode` should do.

ATM AdminState is ignored in polling.

Before
image

After
image

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply 15782`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: tests/data/infinera-groove_4.7.0.json [hunk 1]
```diff
@@ -0,0 +1 @@
+null
```
Comment (Jellyfrog) [inline tests/data/infinera-groove_4.7.0.json:1]: This doesn't look correct :)

**Comment by nickhilliard** [comment]
not quite sure why the existing code isn't working for you. It works fine on FP 4.5.0, e.g.

groove-g30-works-fine

**Comment by Bierchermuesli** [comment]
hey @nickhilliard  thanks for your feedback! that is interesting! we have others G30 chassis with FP4.5.2 and FP4.1.0 with the appearance. Is there a reason you not use cardAdminStatus ?

**Comment by PipoCanaja** [comment]
Hi @Bierchermuesli 
Coud you regenerate tests for other Infinera Groove variants ? 
Thx

**Comment by Bierchermuesli** [comment]
Closed due no interests by the needy

**Pull request closed without merging**